chord_metadata_service/restapi/api_views.py

```diff
@@ -44,173 +47,87 @@ def overview(_request):
-                "count": len(phenotypic_features_counter.keys()),
-                "type": dict(phenotypic_features_counter)
+                "count": phenotypic_features_count,
+                "type": stats_for_field(pheno_models.PhenotypicFeature, "pftype__label")
             },
             "experiments": {
-                "count": len(experiments_set),
-                "study_type": dict(experiments["study_type"]),
-                "experiment_type": dict(experiments["experiment_type"]),
-                "molecule": dict(experiments["molecule"]),
-                "library_strategy": dict(experiments["library_strategy"]),
-                "library_source": dict(experiments["library_source"]),
-                "library_selection": dict(experiments["library_selection"]),
-                "library_layout": dict(experiments["library_layout"]),
-                "extraction_protocol": dict(experiments["extraction_protocol"]),
+                "count": experiments_count,
+                "study_type": stats_for_field(experiments_models.Experiment, "study_type"),
+                "experiment_type": stats_for_field(experiments_models.Experiment, "experiment_type"),
+                "molecule": stats_for_field(experiments_models.Experiment, "molecule"),
+                "library_strategy": stats_for_field(experiments_models.Experiment, "library_strategy"),
+                "library_source": stats_for_field(experiments_models.Experiment, "library_source"),
+                "library_selection": stats_for_field(experiments_models.Experiment, "library_selection"),
+                "library_layout": stats_for_field(experiments_models.Experiment, "library_layout"),
+                "extraction_protocol": stats_for_field(experiments_models.Experiment, "extraction_protocol"),
             },
             "experiment_results": {
-                "count": len(experiment_results_set),
-                "file_format": dict(experiment_results["file_format"]),
-                "data_output_type": dict(experiment_results["data_output_type"]),
-                "usage": dict(experiment_results["usage"])
+                "count": experiment_results_count,
+                "file_format": stats_for_field(experiments_models.ExperimentResult, "file_format"),
+                "data_output_type": stats_for_field(experiments_models.ExperimentResult, "data_output_type"),
+                "usage": stats_for_field(experiments_models.ExperimentResult, "usage")
             },
             "instruments": {
-                "count": len(instrument_set),
-                "platform": dict(instruments["platform"]),
-                "model": dict(instruments["model"])
+                "count": instruments_count,
+                "platform": stats_for_field(experiments_models.Experiment, "instrument__platform"),
+                "model": stats_for_field(experiments_models.Experiment, "instrument__model")
             },
         }
-    })
+    }
+
+    return Response(r)
```

Comment:
Just wondering, the motivation for changing it to a variable instead of a direct response

Reply:
Easier to debug by placing breakpoints. Can be simplified.